raiden/lightclient/handlers/light_client_message_handler.py

```diff
@@ -99,6 +100,18 @@ def is_light_client_protocol_message_already_stored(cls, payment_id: int,
                                               existing_message[7])
         return existing_message
 
+    @classmethod
+    def is_message_already_stored(cls,
```

Comment:
Why was this method necessary to implement the settlement?

Reply:
It's not just for settlement, before this you had duplicated messages since we don't use the payment id anymore, this method checks the type, message content and lc address to ensure that we don't have other message already created, this basically is for when you have the hub down and the you start it again and then it runs all over again and re-creates the message for the LC

Reply:
Ok i have created an issue RLP791 to refactor the method names later.